### PR TITLE
Badge frontend

### PR DESCRIPTION
**Pull request by roger-rangel**
Closes: #5

### CODE REVIEW

File: src/segenie_frontend/src/Hooks/useCreateNFT.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    } catch (error) {
+      console.error(error);
+      throw error;
+    }
+  };
+
+  const newNFT = async (name, description, imageDataURL) => {
+    try {
+      checkAuthToken();
+      const imageIPFSURL = await uploadDataURL(imageDataURL);
```
Comment (Szegoo) [inline src/segenie_frontend/src/Hooks/useCreateNFT.jsx:42]: Why is this ....

File: src/segenie_frontend/src/Hooks/useCreateNFT.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  const fetchAllNFTs = async () => {
+    try {
+      const { hadSuccess, responseResult, errorMessage } =
+        await actor.getAssetViewModelsForFeed({
+          forAuthToken: authToken ?? '0',
+          doSucceed: true,
+        });
+      if (!hadSuccess) throw new Error(errorMessage);
+      const NFTs = responseResult.map((item) => ({
+        id: item.assetId,
```
Comment (Szegoo) [inline src/segenie_frontend/src/Hooks/useCreateNFT.jsx:19]: The `id` is generated automatically on the backend, so the user doesn't have to provide it as input.

File: src/segenie_frontend/src/Hooks/useCreateNFT.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        name: item.assetDisplayName,
+        description: item.assetDescription,
+        imageURL: item.assetSourceImageUrl,
+        comments: item.comments.map((comment) => ({
```
Comment (Szegoo) [inline src/segenie_frontend/src/Hooks/useCreateNFT.jsx:23]: What are these comments going to represent?

File: src/segenie_frontend/src/Hooks/useCreateNFT.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  const newNFT = async (name, description, imageDataURL) => {
+    try {
+      checkAuthToken();
+      const imageIPFSURL = await uploadDataURL(imageDataURL);
+      const { hadSuccess, responseResult, errorMessage } =
+        await actor.mintAsset({
+          forAuthToken: authToken,
+          assetDisplayNameIn: name,
+          assetDescriptionIn: description,
+          assetUrlImageSourceIn: imageIPFSURL,
```
Comment (Szegoo) [inline src/segenie_frontend/src/Hooks/useCreateNFT.jsx:48]: I don't think this has to be an IPFS URL, or is there a reason for this?

File: src/segenie_frontend/src/Views/NewPortal/NewPortal.jsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+import NewPortalFirstModal from './PortalModalComponents/NewPortalFirstModal/NewPortalFirstModal';
+
+//components
+import Layer from '../../components/Layer/Layer';
+import Loader from '../../components/Loader';
+
+const newPortal = () => {
+  const [currentModalIndex, setCurrentModalIndex] = useState(0);
+  const [portal, setPortal] = useState({
+    id: '',
```
Comment (Szegoo) [inline src/segenie_frontend/src/Views/NewPortal/NewPortal.jsx:16]: As I mentioned in the previous comment, the `id` is generated automatically.

File: src/segenie_frontend/src/Views/NewPortal/PortalModalComponents/MintResultCard/MintResultCard.jsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        </div>
+        <Description className="max-h-[150px] overflow-auto">
+          {mintResult.description}
+        </Description>
+      </div>
+    </article>
+  );
+};
+
+// FOR SERGEJ: Not sure if ID Prop Types is correct
```
Comment (Szegoo) [inline src/segenie_frontend/src/Views/NewPortal/PortalModalComponents/MintResultCard/MintResultCard.jsx:28]: The `id` is going to be a number.

File: src/segenie_frontend/src/Hooks/useCreateNFT.jsx (hunk 2 above)
Comment (roger-rangel) [reply src/segenie_frontend/src/Hooks/useCreateNFT.jsx:19]: So I should keep `id: item.assetId` there or delete it?

File: src/segenie_frontend/src/Hooks/useCreateNFT.jsx (hunk 3 above)
Comment (roger-rangel) [reply src/segenie_frontend/src/Hooks/useCreateNFT.jsx:23]: for now we don't need comments for the Portal, I am deleting that

File: src/segenie_frontend/src/Views/NewPortal/NewPortal.jsx (hunk 5 above)
Comment (roger-rangel) [reply src/segenie_frontend/src/Views/NewPortal/NewPortal.jsx:16]: Sounds good, I deleted `id `

File: src/segenie_frontend/src/Hooks/useCreateNFT.jsx (hunk 2 above)
Comment (Szegoo) [reply src/segenie_frontend/src/Hooks/useCreateNFT.jsx:19]: Yes, you should delete it.

**Pull request merged**